examples/bind/ecommerce.amp.html

```diff
@@ -115,15 +183,139 @@
+          </tr>
+        </table>
+      </amp-selector>
+    </div>
+
+    <div class="options price">
+      <h6>PRICE : <span [text]="shirts[selected.id].sizes[selectedSize] || '---'">---</span></h6>
+    </div>
+
+    <div class="options sizes">
+      <h6>SIZE :</h6>
+
+      <amp-selector on="select:AMP.setState({selectedSize: event.targetOption})">
+        <table>
+          <tr>
+            <td [class]="!shirts[selected.id].sizes['XS'] ? 'unavailable' : ''">
+              <div option="XS">XS</div>
+            </td>
+            <td [class]="!shirts[selected.id].sizes['S'] ? 'unavailable' : ''">
+              <div option="S">S</div>
+            </td>
+            <td [class]="!shirts[selected.id].sizes['M'] ? 'unavailable' : ''"
+                class="unavailable">
+              <div option="M">M</div>
+            </td>
+            <td [class]="!shirts[selected.id].sizes['L'] ? 'unavailable' : ''"
+                class="unavailable">
+              <div option="L">L</div>
+            </td>
+            <td [class]="!shirts[selected.id].sizes['XL'] ? 'unavailable' : ''"
+                class="unavailable">
+              <div option="XL">XL</div>
+            </td>
+          </tr>
+        </table>
+      </amp-selector>
+    </div>
+
+    <div class="options quantity">
+      <h6>QUANTITY :</h6>
+      <select [disabled]="!shirts[selected.id].sizes[selectedSize]">
+        <option value="1">1</option>
+        <option value="2">2</option>
+        <option value="3">3</option>
+      </select>
+    </div>
+
+    <div class="options purchase">
+      <h5 [text]="(selectedSize && shirts[selected.id].sizes)
+            ? (shirts[selected.id].sizes[selectedSize] ? 'In stock' : 'Out of stock')
+            : '---'"
+          [class]="(!selectedSize || shirts[selected.id].sizes[selectedSize])
+            ? 'mdl-color-text--grey'
+            : 'mdl-color-text--red'">
+        ---</h5>
+
+      <!--
+      TODO(choumx):
+      - A nice way to DRY expressions that's less powerful than allowing chained AMP.setState()?
+      - Would users assume that chaining <amp-variable> is possible? Probably...
+
+      <amp-variable
+        id="showAddToBag"
+        value="!selectedSize || !shirts[selected.id].sizes || shirts[selected.id].sizes[selectedSize]">
+      </amp-variable>
+      -->
+
+      <button class="mdl-button mdl-button--raised mdl-button--accent"
+          [class]="'mdl-button mdl-button--raised mdl-button--accent ' +
+            ((!selectedSize || !shirts[selected.id].sizes || shirts[selected.id].sizes[selectedSize]) ? '' : 'hidden')"
+          [disabled]="!selectedSize || !shirts[selected.id].sizes" disabled>
+        ADD TO BAG
+      </button>
+      <button class="mdl-button mdl-button--raised hidden"
+          [class]="'mdl-button mdl-button--raised ' +
+            ((!selectedSize || !shirts[selected.id].sizes || shirts[selected.id].sizes[selectedSize]) ? 'hidden' : '')">
```

Comment:
Could you add a comment as to when this button is supposed to be shown vs hidden?